pkg/translate/field_mask_test.go

```diff
@@ -0,0 +1,92 @@
+package translate
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/stretchr/testify/assert"
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
+)
+
+func Test_ValidateFieldMask(t *testing.T) {
```

Comment:
Although these tests may seem excessive since a 3rd party package is being used and it likely has its own tests, they were originally implemented to loop through all fields of a proto message using reflection. Later on, it was discovered that the package has useful functions for this purpose. Despite this, the decision was made to keep the tests as they won’t cause any harm.